SpatialGDK/Source/SpatialGDK/Private/Tests/LoadBalanceEnforcerTest.cpp

```diff
@@ -0,0 +1,358 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "EngineClasses/SpatialLoadBalanceEnforcer.h"
+#include "EngineClasses/SpatialVirtualWorkerTranslator.h"
+#include "Interop/SpatialStaticComponentView.h"
+#include "Schema/AuthorityIntent.h"
+#include "Tests/TestingSchemaHelpers.h"
+
+#include "CoreMinimal.h"
+
+#include "Schema/ClientEndpoint.h"
+#include "SpatialConstants.h"
+#include "SpatialView/ComponentData.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/ViewDelta.h"
+#include "Tests/SpatialView/SpatialViewUtils.h"
+#include "Utils/SchemaOption.h"
+
+#define LOADBALANCEENFORCER_TEST(TestName) GDK_TEST(Core, SpatialLoadBalanceEnforcer, TestName)
+
+// Test Globals
+namespace
+{
+const PhysicalWorkerName UnrealWorker = TEXT("UnrealWorker");
+const WorkerRequirementSet UnrealRequirementSet = WorkerRequirementSet{ WorkerRequirementSet{ WorkerAttributeSet{ UnrealWorker } } };
```

Comment:
It looks like there's one too many WorkerRequirementSets here

Reply:
Few places in this file

Reply:
wow I have copy pasted it everywhere- ty

Reply:
find and replaced